pkg/controlplane/controller/crdregistration/crdregistration_controller.go

```diff
@@ -198,6 +205,18 @@ func (c *crdRegistrationController) handleVersionUpdate(groupVersion schema.Grou
 	if err != nil {
 		return err
 	}
+
+	//!TODO: This is likely not the best way to accomplish this.
+	// crdAPIHandler giving error 500 for missing RequestInfo?
+	hash, err := discovery.GetGroupVersionHash(
+		"/apis/"+groupVersion.Group+"/"+groupVersion.Version,
+		genericapifilters.WithRequestInfo(c.crdAPIHandler, &request.RequestInfoFactory{}),
```

Comment:
I'm very skeptical of WithRequestInfo being needed here? Is this the right way to do this?